cosi/threshold/types/mod.go

```diff
@@ -149,6 +151,14 @@ func (s *Signature) Equal(o crypto.Signature) bool {
 	return ok && other.agg.Equal(s.agg) && bytes.Equal(s.mask, other.mask)
 }
 
+// String implements fmt.Stringer. It returns a string representation of the
+// signature.
+func (s *Signature) String() string {
+	mask := new(big.Int).SetBytes(s.mask)
+
+	return fmt.Sprintf("thres[%b]:%s", mask, s.agg)
```

Comment:
why just not `fmt.Sprintf("thres[%b]:%s", s.mask, s.agg)` ?

Reply:
Unfortunately, it prints in an array style like `[0000] [0010]` and I don't like that ;)